File: App.Native01/app/src/main/java/br/com/ikecode/tockup/models/Message.java
package br.com.ikecode.tockup.models;

/**
 * Created by Leandro Barral on 05/12/2016.
 * Intellectual Property of "IkeCode {SmartSolutions}"
 */

public class Message {
    public int fromUserId;
    public int toUserId;
    public int status;
    public int selectedPrefixId;
    public int selectedSuffixId;
    public int messageType;
    public int languageId;
    public User fromUser;
    public MessagePrefix selectedPrefix;
    public MessageSuffix selectedSuffix;
    public User toUser;
}

File: Model/Message.cs
namespace PassarinhoContou.Model
{
    public partial class Message : BaseModel
    {
        public int FromUserId { get; set; }
        public int ToUserId { get; set; }
        public MessageStatusEnum Status { get; set; }
        public int SelectedPrefixId { get; set; }
        public int SelectedSuffixId { get; set; }
        public MessageTypeEnum MessageType { get; set; }
        public int LanguageId { get; set; }
        
        public virtual User FromUser { get; set; }
        public virtual MessagePrefix SelectedPrefix { get; set; }
        public virtual MessageSuffix SelectedSuffix { get; set; }
        public virtual User ToUser { get; set; }
    }
}
